Makefile

```diff
@@ -9,3 +9,10 @@ clean:
 
 e2e:
 	go test -v --count=1 --tags=e2e ./e2e
+
+push:
+	echo "$(shell cat devenv/state/registry.txt)/app-routing-operator:$(shell date +%s)" > devenv/state/operator-image-tag.txt
```

Comment:
Where can we get this registry.txt file? create the file locally before running push command?

Reply:
yes, it's created by terraform when creating the environment `make dev`